pulp_container/tests/functional/api/test_push_content.py

```diff
@@ -273,6 +292,9 @@ def test_push_private_repository(
     Test that the same user can pull, but another cannot.
     Test that the other user can pull after marking it non-private.
     """
+    if settings.TOKEN_AUTH_DISABLED:
```

Comment:
what about testing helpless_user pull of private/public repo?

Reply:
I extended the `test_push_with_no_perms` test case to include pulling of a private repository.

Reply:
where?

Reply:
I removed it after you opened the new issue: https://github.com/pulp/pulp_container/pull/1649#discussion_r1619075596.

Reply:
the pull of private repo will pass, so you can add it back. it's the catalog list of private repos that will fail. or you decided not to add the test unless we figure out whether it is ok for anon user to be able to pull private repos with token auth disabled?

Reply:
I was still thinking whether it is right to completely ignore the private flag even though the token authentication is disabled. I have just added back the change. I am not catching an exception anymore, just verifying if using the private flag does not have any impact on pulling.